x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/agent_list_page/index.tsx

```diff
@@ -162,30 +281,48 @@ export const AgentListPage: React.FunctionComponent<{}> = () => {
         return null;
       },
     },
+    {
+      field: 'active',
+      name: i18n.translate('xpack.ingestManager.agentList.statusColumnTitle', {
+        defaultMessage: 'Status',
+      }),
+      truncateText: true,
```

Comment:
Note that `truncateText` does not work when you have a `render()` function defined for the column. I found out about this a few weeks ago and ended up creating a Component to [truncate it myself](https://github.com/elastic/kibana/blob/fe21356020ee9567e3164869bae768c1220db130/x-pack/plugins/endpoint/public/applications/endpoint/components/truncate_text.ts#L9).